zerver/tests/tests.py

```diff
@@ -74,6 +74,9 @@ def find_dict(lst, k, v):
             return dct
     raise Exception('Cannot find element in list where key %s == %s' % (k, v))
 
+# same as in test_uploads.py
+TEST_AVATAR_DIR = os.path.join(os.path.dirname(__file__), 'images')
```

Comment:
I'd like to punt on fixing the duplication for this particular series.  Really, instead of putting TEST_AVATAR_DIR in test_helpers, we should probably have a helper at a slightly higher level of abstraction, such as get_test_image_file().  But introducing that would touch some other tests and complicate the immediate goal of just getting coverage.